### PR TITLE
Feat: optimize imports packages to reduce 75% cpu with better performance

### PR DESCRIPTION
**Pull request by FogDong**
Signed-off-by: FogDong <dongtianxin.tx@alibaba-inc.com>

### Description of your changes

optimize imports packages

Before:
image

After:
image

I have:

- [ ] Read and followed KubeVela's [contribution process](https://github.com/kubevela/kubevela/blob/master/contribute/create-pull-request.md).
- [ ] [Related Docs](https://github.com/kubevela/kubevela.io) updated properly. In a new feature or configuration option, an update to the documentation is necessary. 
- [ ] Run `make reviewable` to ensure this PR is ready for review.
- [ ] Added `backport release-x.y` labels to auto-backport this PR if necessary.

### CODE REVIEW

File: pkg/stdlib/packages.go [hunk 1]
```diff
@@ -63,26 +64,43 @@ func GetPackages(tagTempl string) (map[string]string, error) {
...
 	return map[string]string{
-		"vela/op": opContent + "\n" + tagTempl,
-		"vela/ql": qlContent + "\n" + tagTempl,
+		"vela/op": opContent,
+		"vela/ql": qlContent,
 	}, nil
 }
 
 // AddImportsFor install imports for build.Instance.
 func AddImportsFor(inst *build.Instance, tagTempl string) error {
```
Comment (wonderflow) [inline pkg/stdlib/packages.go:73]: when will this func be used?

File: pkg/stdlib/packages.go [hunk 2]
```diff
@@ -63,26 +64,43 @@ func GetPackages(tagTempl string) (map[string]string, error) {
...
+		if err := p.AddFile("-", tagTempl); err != nil {
+			return err
+		}
+		inst.Imports = append(inst.Imports, p)
+	}
+	return nil
+}
+
+// InitBuiltinImports init builtin imports
+func InitBuiltinImports() ([]*build.Instance, error) {
```
Comment (wonderflow) [inline pkg/stdlib/packages.go:89]: Let's use `init()` for this feature

File: pkg/workflow/tasks/custom/task.go [hunk 3]
```diff
@@ -278,18 +278,18 @@ func buildValueForStatus(ctx wfContext.Context, step v1beta1.WorkflowStep, pd *p
...
 	contextTempl := getContextTemplate(ctx, id, pCtx)
-	return value.NewValue(templ+contextTempl, pd, contextTempl, value.ProcessScript, value.TagFieldOrder)
+	return value.NewValue(templ+contextTempl, pd, "", value.ProcessScript, value.TagFieldOrder)
 }
 
 // MakeValueForContext makes context value
 func MakeValueForContext(ctx wfContext.Context, pd *packages.PackageDiscover, id string, pCtx process.Context) (*value.Value, error) {
 	contextTempl := getContextTemplate(ctx, id, pCtx)
-	return value.NewValue(contextTempl, pd, contextTempl)
+	return value.NewValue(contextTempl, pd, "")
```
Comment (wonderflow) [inline pkg/workflow/tasks/custom/task.go:292]: why they're not needed now? and above

File: pkg/stdlib/packages.go (hunk 1 above)
Comment (FogDong) [reply pkg/stdlib/packages.go:77]: When you want to new a CUE Value.

File: pkg/workflow/tasks/custom/task.go (hunk 3 above)
Comment (FogDong) [reply pkg/workflow/tasks/custom/task.go:292]: If you add tag in `NewValue`, eg.`context: "test"`, you can use `op.context` to refer to `test`, we're not using this feature right now.

File: pkg/stdlib/packages_test.go [hunk 4]
```diff
@@ -36,8 +36,8 @@ func TestGetPackages(t *testing.T) {
 
 	builder := &build.Instance{}
 	builder.AddFile("-", `
-import "vela/op"
-out: op.context`)
+import "vela/custom"
+out: custom.context`)
```
Comment (wonderflow) [inline pkg/stdlib/packages_test.go:40]: does it mean the API changed?

Comment (FogDong) [reply pkg/stdlib/packages_test.go:40]: Actually I don't think we're using this feature right now.

**Review by wonderflow — APPROVED** [review]

**Review by Somefive — APPROVED** [review]

**Pull request merged**